### PR TITLE
Check if baseUrl has a last / and remove if necessary

### PR DESCRIPTION
**Pull request by patrickemuller**
Also add Docker files

### CODE REVIEW

File: src/apiAction.js [hunk 1]
```diff
@@ -99,8 +100,7 @@ const dispatchAction = async (dispatch, { debugRequests, ...options }) => {
   const { baseUrl, path, parentId, prefix, requestAttributes } = options
 
   dispatch({ parentId, type: `${prefix}_REQUEST`, ...requestAttributes })
-
-  const url = `${baseUrl}${path}`
+  const url = `${replace(baseUrl, /\/$/, '')}${path}`
```
Comment (giovanisleite) [inline src/apiAction.js]: This way we fix the case that the `/` it is on `baseUrl`, `path` (path build by croods has `/`, but user can set it too.) or missing `/` in both.
```suggestion
  const url = replace(`${baseUrl}/${path}`, /[\/]{2,}/, '/')
```

What do you think?

Comment (patrickemuller) [reply src/apiAction.js]: Nice!

**Comment by gustavoguichard** [comment]
Can't see why `Dockerfile` is part of this PR 😬

**Comment by patrickemuller** [comment]
> Can't see why `Dockerfile` is part of this PR 😬

Ooops, my mistake, gonna create another PR implementing Docker support

**Pull request merged**